Dictionaries/hrv_OCRFixReplaceList.xml

```diff
@@ -573,6 +574,7 @@
     <RegEx find="([bB])ožij([aeiu]|e[mg]|om|ima)?\b" replaceWith="$1ožj$2" />
     <RegEx find="boži[čć]([aeiu]|em|ima)?\b" replaceWith="Božić$1" />
     <RegEx find="(?&lt;!\A|[.!?][&quot;”’]?\s+)\bBoži[čć]n([aeiou]|om|im)\b" replaceWith="božićn$1" />
+    <RegEx find="([bB])ubašvab" replaceWith="žohar" />
```

Comment:
Perhaps, `$1` is missing in `replaceWith`? (`<RegEx find="([bB])ubašvab" replaceWith="$1žohar" />`)